moveit_core/utils/CMakeLists.txt

```diff
@@ -1,9 +1,21 @@
 set(MOVEIT_LIB_NAME moveit_utils)
 
-add_library(${MOVEIT_LIB_NAME} src/lexical_casts.cpp)
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+if(CATKIN_ENABLE_TESTING)
+  add_library(${MOVEIT_LIB_NAME}
```

Comment:
i don't like how this library changes scope based on whether testing is enabled or not. why not just have a separate library for things that are only for tests, and other utils?